From 3f7225d3ce4640c7549975f8ba233b69b2029f6a Mon Sep 17 00:00:00 2001
From: kbandi <kushal.bandi@intel.com>
Date: Thu, 28 Jun 2018 21:30:39 -0700
Subject: [PATCH 1113/1240] intel-ipu4: Reversing Fixed compilation error
 without DEBUG_FS patch

[Reason]
	Patch needs to be checked in to domain maintainer trunk &
	not mainline lts branch directly

Change-Id: If5df1e74126d67c62d72321d3b312482e8a8c385
Signed-off-by: kbandi <kushal.bandi@intel.com>
---
 drivers/media/pci/intel/ipu-buttress.c | 25 ++++++++++++-------------
 1 file changed, 12 insertions(+), 13 deletions(-)

diff --git a/drivers/media/pci/intel/ipu-buttress.c b/drivers/media/pci/intel/ipu-buttress.c
index 19c64f1..0f128f3 100644
--- a/drivers/media/pci/intel/ipu-buttress.c
+++ b/drivers/media/pci/intel/ipu-buttress.c
@@ -1432,19 +1432,6 @@ int ipu_buttress_tsc_read(struct ipu_device *isp, u64 *val)
 }
 EXPORT_SYMBOL_GPL(ipu_buttress_tsc_read);
 
-u64 ipu_buttress_tsc_ticks_to_ns(u64 ticks)
-{
-	/*
-	 * TSC clock frequency is 19.2MHz,
-	 * converting TSC tick count to ns is calculated by:
-	 * ns = ticks * 1000 000 000 / 19.2Mhz
-	 *    = ticks * 1000 000 000 / 19200000Hz
-	 *    = ticks * 10000 / 192 ns
-	 */
-	return ticks * 10000 / 192;
-}
-EXPORT_SYMBOL_GPL(ipu_buttress_tsc_ticks_to_ns);
-
 #ifdef CONFIG_DEBUG_FS
 
 static int ipu_buttress_reg_open(struct inode *inode, struct file *file)
@@ -1510,6 +1497,18 @@ static int ipu_buttress_start_tsc_sync_set(void *data, u64 val)
 DEFINE_SIMPLE_ATTRIBUTE(ipu_buttress_start_tsc_sync_fops, NULL,
 			ipu_buttress_start_tsc_sync_set, "%llu\n");
 
+u64 ipu_buttress_tsc_ticks_to_ns(u64 ticks)
+{
+	/*
+	 * TSC clock frequency is 19.2MHz,
+	 * converting TSC tick count to ns is calculated by:
+	 * ns = ticks * 1000 000 000 / 19.2Mhz
+	 *    = ticks * 1000 000 000 / 19200000Hz
+	 *    = ticks * 10000 / 192 ns
+	 */
+	return ticks * 10000 / 192;
+}
+EXPORT_SYMBOL_GPL(ipu_buttress_tsc_ticks_to_ns);
 
 static int ipu_buttress_tsc_get(void *data, u64 *val)
 {
-- 
2.7.4

